app/src/components/template/TemplateObservationList.tsx

```diff
@@ -77,33 +77,39 @@ const TemplateObservationsList: React.FC<ITemplateObservationsListProps> = (prop
               </TableRow>
             </TableHead>
             <TableBody>
-              {props.attachmentsList.length > 0 &&
-                props.attachmentsList.slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage).map((row, index) => (
-                  <TableRow key={row.id}>
-                    <TableCell component="th" scope="row">
-                      <Link underline="always" component="button" variant="body2" onClick={() => viewFileContents(row)}>
-                        {row.fileName}
-                      </Link>
-                    </TableCell>
-                    <TableCell>{getFormattedDate(DATE_FORMAT.ShortMediumDateTimeFormat, row.lastModified)}</TableCell>
-                    <TableCell>{getFormattedFileSize(row.size)}</TableCell>
-                  </TableRow>
-                ))}
-              {!props.attachmentsList.length && (
+              {props.templateObservationsList.length > 0 &&
+                props.templateObservationsList
+                  .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
+                  .map((row, index) => (
+                    <TableRow key={row.id}>
+                      <TableCell component="th" scope="row">
+                        <Link
+                          underline="always"
+                          component="button"
+                          variant="body2"
+                          onClick={() => viewFileContents(row)}>
+                          {row.fileName}
+                        </Link>
+                      </TableCell>
+                      <TableCell>{getFormattedDate(DATE_FORMAT.ShortMediumDateTimeFormat, row.lastModified)}</TableCell>
+                      <TableCell>{getFormattedFileSize(row.size)}</TableCell>
+                    </TableRow>
+                  ))}
+              {!props.templateObservationsList.length && (
```

Comment:
I think this may print out 0 onto the screen...
might need to do:
```
{props.templateObservationsList.length === 0 && (
```

Reply:
if I am mistaken and it doesnt, then this is all good 😄

Reply:
ya thats a funny issue.  Not quite sure when it happens, but Im going to take a guess and say it wont happen in this case.  Mainly Im thinking that the `!` will turn it into a boolean, and so wont print?  Unless it prints 'false' or something haha

Reply:
Looked in openshift, works as expected, no issues

Reply:
yeah, it works as expected, but I changed it to `{props.templateObservationsList.length === 0 && (` anyway to avoid issues down the line.